debezium-server/debezium-server-pubsub/src/main/java/io/debezium/server/pubsub/PubSubChangeConsumer.java

```diff
@@ -54,6 +64,37 @@
 
     private static final String PROP_PREFIX = "debezium.sink.pubsub.";
     private static final String PROP_PROJECT_ID = PROP_PREFIX + "project.id";
+    private static final String PROP_BATCH_DELAY_THRESHOLD = PROP_PREFIX + "batch.settings.setDelayThreshold";
+    private static final String PROP_BATCH_BUFFER_SIZE = PROP_PREFIX +  "batch.settings.setElementCountThreshold";
+    private static final String PROP_BATCH_BUFFER_BYTES = PROP_PREFIX +  "batch.settings.setRequestByteThreshold";
+
+    private static final String PROP_FLOW_CONTROL_ENABLED = PROP_PREFIX + "flowControl.enabled";
+    private static final String PROP_FLOW_CONTROL_OUTSTANDING_MESSAGES = PROP_PREFIX +  "flowControl.settings.maxOutstandingRequestMessages";
+    private static final String PROP_FLOW_CONTROL_OUTSTANDING_BYTES = PROP_PREFIX +  "flowControl.settings.maxOutstandingRequestBytes";
+
+    private static final String PROP_RETRY_TOTAL_TIMEOUT = PROP_PREFIX +  "retry.settings.setTotalTimeout";
+    private static final String PROP_RETRY_MAX_RPC_TIMEOUT = PROP_PREFIX +  "retry.settings.setMaxRpcTimeout";
+    private static final String PROP_RETRY_INITIAL_DELAY = PROP_PREFIX +  "retry.settings.setInitialRetryDelay";
+    private static final String PROP_RETRY_DELAY_MULTIPLIER = PROP_PREFIX +  "retry.settings.setRetryDelayMultiplier";
+    private static final String PROP_RETRY_MAX_DELAY = PROP_PREFIX +  "retry.settings.setMaxRetryDelay";
+    private static final String PROP_RETRY_INITIAL_RPC_TIMEOUT = PROP_PREFIX +  "retry.settings.setInitialRpcTimeout";
+    private static final String PROP_RETRY_RPC_TIMEOUT_MULTIPLIER = PROP_PREFIX +  "retry.settings.setRpcTimeoutMultiplier";
+
+    private Integer maxDelayThresholdMs;
+    private Long maxBufferSize;
+    private Long maxBufferBytes;
+
+    private boolean flowControlEnabled;
+    private Long maxOutstandingMessages;
+    private Long maxOutstandingRequestBytes;
+
+    private Integer maxTotalTimeoutMs;
+    private Integer maxRequestTimeoutMs;
+    private Integer initialRetryDelay;
+    private double retryDelayMultiplier;
+    private Long maxRetryDelay;
+    private Integer initialRpcTimeout;
+    private double rpcTimeoutMultiplier;
```

Comment:
Could we obtain them via injection instead (as e.g. the case for `ordering.enabled`)? Should be a bit more concise?

Reply:
Done.  I wasn't able to have the default be Long.MAX_VALUE using injection though so just used the actual Long.MAX_VALUE in the default if that makes sense.  Seems to work.